client/shipping-label/state/normalize-address.js

```diff
@@ -31,6 +31,10 @@ export default ( dispatch, address, type, addressNormalizationURL, nonce ) => {
 				setTimeout( () => resolve( ! error ), 0 );
 			}
 		};
-		saveForm( setIsSaving, setSuccess, _.noop, setError, addressNormalizationURL, nonce, 'POST', { address, type } );
+		saveForm( setIsSaving, setSuccess, _.noop, setError, addressNormalizationURL, nonce, 'POST', { orderId, address, type } );
 	} );
 };
+
+export const acceptAddress = ( dispatch, orderId, address, type, addressAcceptURL, nonce ) => {
+	saveForm( _.noop, _.noop, _.noop, _.noop, addressAcceptURL, nonce, 'POST', { orderId, address, type } );
```

Comment:
This kind of function call to `saveForm` hurts my brain (all those noop...). This is a good candidate for a refactor in a near future. Maybe refactor `saveForm` to use Promises, or use something like Node's `EventEmitter` pattern.